parlai/core/teachers.py

```diff
@@ -2514,6 +2514,11 @@ def reset(self):
             )  # reset the count of samples loaded
             self._enqueue_request()
 
+    def shutdown(self):
+        # self._drain(self.chunks)
```

Comment:
why did you comment this out?? are you sure you want to do this?

Reply:
It's not necessary, need to delete